src/openforms/registrations/contrib/stuf_zds/tests/test_backend.py

```diff
@@ -3465,3 +3472,249 @@ def test_register_submission_with_payment(self):
                     f"//stuf:extraElementen/stuf:extraElement[@naam='{name}']",
                     value,
                 )
+
+
+class StufZDSPluginPartnersComponentVCRTests(OFVCRMixin, StUFZDSTestBase):
+    VCR_TEST_FILES = TESTS_DIR / "files"
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.plugin = StufZDSRegistration(PLUGIN_IDENTIFIER)
+        cls.zds_service = StufServiceFactory.create(
+            soap_service__url="http://localhost/stuf-zds"
+        )
+        config = StufZDSConfig.get_solo()
+        config.service = cls.zds_service
+        config.zaakbetrokkene_partners_omschrijving = "A description"
+        config.save()
+        cls.addClassCleanup(StufZDSConfig.clear_cache)
+
+        cls.options: RegistrationOptions = {
+            "zds_zaaktype_code": "foo",
+            "zds_documenttype_omschrijving_inzending": "foo",
+            "zds_zaakdoc_vertrouwelijkheid": "GEHEIM",
+        }
+
+        hc_config = HaalCentraalConfig(
+            brp_personen_service=ServiceFactory.build(
+                api_root="http://localhost:5010/haalcentraal/api/brp/"
+            ),
+            brp_personen_version=BRPVersions.v20,
+        )
+        hc_config_patcher = patch(
+            "openforms.contrib.haal_centraal.clients.HaalCentraalConfig.get_solo",
+            return_value=hc_config,
+        )
+
+        global_config = GlobalConfiguration(
+            family_members_data_api=FamilyMembersDataAPIChoices.haal_centraal
+        )
+        global_config_patcher = patch(
+            "openforms.config.models.GlobalConfiguration.get_solo",
+            return_value=global_config,
+        )
+
+        hc_config = hc_config_patcher.start()
+        global_config = global_config_patcher.start()
+        cls.addClassCleanup(hc_config_patcher.stop)
+        cls.addClassCleanup(global_config_patcher.stop)
+
+    def test_create_zaak_with_partners_as_betrokkene(self):
```

Comment:
can you also assert that it's not pesent in extra elementen, both the `partnersKey` and `partners_immutable`?